pkgs/development/python-modules/django-anymail/default.nix

```diff
@@ -0,0 +1,37 @@
+{ 
+    stdenv,
+    buildPythonPackage,
+    fetchPypi,
+    six,
+    requests,
+    django,
+}:
+
+buildPythonPackage rec {
+  pname = "django-anymail";
+  version = "6.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1a3hqchi1kl12522fz2dxvym1j9n33d9ahlwzxqnf0q9w1frqz4l";
+  };
+
+  propagatedBuildInputs = [
+    six
+    requests
+    django
+  ];
+
+  doCheck = false;
+
+  # this package allows multiple email backends
+  # but each of those backends depends on a python package
+  # which are currently missing from the repositories
+  # only mailgun works by default
```

Comment:
We certainly have `boto3` which will enable SES.

Reply:
:+1:  thanks